nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -230,6 +230,12 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.mockftpserver</groupId>
```

Comment:
I checked the [project homepage](http://mockftpserver.sourceforge.net/) to check for the license to see if `LICENSE` or `NOTICE` needed to be updated, but I don't see a license posted. Do you know which it's published under?